File: crack.py
words = open('WordList.txt', 'r')
valid_handles = sorted([word.replace('at', '@').rstrip() for word in words if 'at' in word[:2] if "'" not in word], lambda x,y: cmp(len(x), len(y)))
print 'Short handles'
for i in range(0, 20):
  print valid_handles[i] + ' : ' + valid_handles[i].replace('@', 'at')

print '\nLong handles'
for i in range(0, 20):
  i += 1
  print valid_handles[-i] + ' : ' + valid_handles[-i].replace('@', 'at')

File: crack.js
var Lazy = require("lazy");
var fs = require('fs');
var async = require("async");
var LazyJs = require("lazy.js");



words = new Lazy(fs.createReadStream(process.argv[2]))
    .lines
    .map(String)
    .filter(function(line) {
        if (/^at/.test(line) && !/'/.test(line)) {

            return true;
        }else{
            return false;
        }
    })
    .map(function(line) {
        return line.replace(/at/g, '@').trimRight();
    })
    .join(function(words){
        var sequence = LazyJs(words).sortBy(function(word){
            return word.length;
        }) ;
        console.log("Short Handle");
        sequence.first(20).each(function(word){
            console.log(word + ":" + word.replace(/@/g, 'at'));
        });

        console.log("Long Handles");
        sequence.last(20).each(function(word){
            console.log(word + ":" + word.replace(/@/g, 'at'));
        });
    });